cmd/root.go

```diff
@@ -11,21 +13,34 @@ import (
 )
 
 var (
-	debug bool
-	quiet bool
+	debug      bool
+	quiet      bool
+	cpuprofile string
 )
 
 func init() {
 	rootCmd.PersistentFlags().BoolVar(&debug, "debug", false, "enable verbose output")
 	rootCmd.PersistentFlags().BoolVarP(&quiet, "quiet", "q", false, "do not output to console; use return code to determine success/failure")
-
+	rootCmd.PersistentFlags().StringVarP(&cpuprofile, "cpuprofile", "p", "", "activate cpu profiling via pprof and write to file")
 	rootCmd.AddCommand(exp.ExpCmd)
 }
 
 var rootCmd = &cobra.Command{
 	Use:          "fogg",
 	Short:        "",
 	SilenceUsage: true,
+	PersistentPreRun: func(cmd *cobra.Command, args []string) {
+		fmt.Printf("Inside rootCmd PersistentPreRun with args: %v\n", args)
```

Comment:
what's the reasoning between fmt.Printf vs log.debug vs logrus?

Reply:
Not sure I have a good reason, this code is copy/paste.

Reply:
I will look at doing another PR to standardize.

Reply:
is it worth choosing for consistency?

Reply:
Yeah, probably should do logrus for everything.